Rubberduck.VBEEditor/Extensions/ComponentTypeExtensions.cs

```diff
@@ -1,76 +1,103 @@
 using System.Collections.Generic;
+using System.Linq;
 using Rubberduck.VBEditor.SafeComWrappers;
 
 namespace Rubberduck.VBEditor.Extensions
 {
     public static class ComponentTypeExtensions
     {
+        //VBA and VB6
         public const string ClassExtension = ".cls";
         public const string FormExtension = ".frm";
         public const string StandardExtension = ".bas";
-        public const string FormBinaryExtension = ".frx";
+
+        //VBA only
         public const string DocClassExtension = ".doccls";
 
-        //TODO: double check whether the guesses below are correct.
+        //VB6 only
         public const string UserControlExtension = ".ctl";
         public const string PropertyPageExtension = ".pag";
         public const string DocObjectExtension = ".dob";
+        public const string ActiveXDesignerExtension = ".dsr";
+        public const string ResourceExtension = ".res";
+
+        //Binary resources
```

Comment:
This might be pedantic but I'd call it `VB6 Binary resources` to remind that it's still VB6 only.

Reply:
`.frx` is not VB6 only. It is also used for the binary file of user forms.